src/printer.js

```diff
@@ -1834,8 +1841,9 @@ function genericPrintNoParens(path, options, print) {
 function printStatementSequence(path, options, print) {
   let printed = [];
 
-  path.map(function(stmtPath, i) {
-    var stmt = stmtPath.getValue();
+  path.map(stmtPath => {
```

Comment:
You should send a separate pull request to remove the i, not add it to a random one :)